python/paddle/fluid/tests/unittests/test_parallel_executor.py

```diff
@@ -0,0 +1,130 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.

```

Comment:
The test name, `test_parallel_executor.py`, is confusing.
Why do you add `test_parallel_executor.py`?

Reply:
Just for the coverage test purpose, I didnt find a file cover all the flags regarding to parallel_executor.